deployment-scripts/helm-charts/deepfence-console/templates/ingress/ingress.yaml

```diff
@@ -1,23 +1,24 @@
-{{- if eq "true" .Values.saas_deployment }}
+{{- if .Values.saas_deployment }}
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
     alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
     alb.ingress.kubernetes.io/backend-protocol: HTTPS
-    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-1:122565780891:certificate/46d330cf-126f-49e4-9866-82fc97492f67
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}, {"HTTP":80}]'
     alb.ingress.kubernetes.io/scheme: internet-facing
-    alb.ingress.kubernetes.io/tags: Environment=prod
+    alb.ingress.kubernetes.io/certificate-arn: {{ .Values.cert_arn }}
     alb.ingress.kubernetes.io/target-type: ip
     alb.ingress.kubernetes.io/group.name: deepfence-1
```

Comment:
can we make this a variable ? this is aws target group for loadbalancer

Reply:
This annotation is not being set in the latest version
```
alb.ingress.kubernetes.io/tags: Environment=prod
```

Reply:
tags maybe not required